titles/aap-on-gcp/aap-on-gcp.asciidoc

```diff
@@ -10,18 +10,18 @@
 include::attributes/attributes.adoc[]
 
 // Book Title
-= {PlatformName} from GCP guide
+= {AAPonGCP} guide
```

Comment:
Suggestion: Avoid using attributes in the title: it must match the docinfo.xml title exactly, so if the attribute changes, the docs build will fail.